modules/tinymce/src/core/test/ts/browser/keyboard/PageUpDownKeysTest.ts

```diff
@@ -50,7 +51,7 @@ describe('browser.tinymce.core.keyboard.PageUpDownKeyTest', () => {
   });
 
   context('Page Down', () => {
-    it('caret is at the beginning of an inline element, escape outside the node', () => {
+    it('Caret should be placed before the inline element if it is at the start of the inline element', () => {
```

Comment:
Shouldn't these be after, Since you are moving down?

- [ ] Double check the test sentences

Reply:
Done